test/test_gh_pages.py

```diff
@@ -8,6 +8,7 @@
 from . import tools
 
 
+@pytest.mark.skipif(tools.HAS_PYPY, reason="Flaky on pypy")
```

Comment:
Am I missing something about this test, or should this not be checking whether you are actually _testing on pypy_ rather than whether it just happens to be installed?